tests/integration/run.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-# List of modules to be tested
-TESTS=""
+# List of the currently working tests
```

Comment:
rebase issue ?

Reply:
clearly

Reply:
shouldn't be a problem since the TESTS variable is replaced